components/server/ee/src/billing/billing-mode.ts

```diff
@@ -166,15 +169,50 @@ export class BillingModesImpl implements BillingModes {
             return usageBased();
         }
         if (hasCbPaidTeamMembership || hasCbPaidTeamSeat || canUpgradeToUBB) {
-            // TODO(gpl): Q: How to test the free-tier, then? A: Make sure you have no CB paid seats anymore
-            // For that we could add a new field here, which lists all seats that are "blocking" you, and display them in the UI somewhere.
-            return { mode: "chargebee", canUpgradeToUBB: true }; // UBB is enabled, but no seat nor subscription yet.
+            // Q: How to test the free-tier, then? A: Make sure you have no CB paid seats anymore
+            // For that we lists all Team Subscriptions/Team Memberships that are "blocking" you, and display them in the UI somewhere.
+            const result: BillingMode = { mode: "chargebee", canUpgradeToUBB }; // UBB is enabled, but no seat nor subscription yet.
+
+            const teamNames = [];
+            for (const tm of teamsModes) {
+                if (tm.mode === "chargebee" && tm.teamNames) {
+                    teamNames.push(`Team Membership: ${tm.teamNames}`);
```

Comment:
**issue(non-blocking)**: Could we somehow, serialize the team names with a comma instead of repeating the prefix? Like _Team A, Team B_.

| Plans | Admin |
|-|-|
| <img width="383" alt="Screenshot 2022-10-27 at 2 25 02 PM" src="https://user-images.githubusercontent.com/120486/198272465-95a6476c-959d-450c-be3f-e22c182e879f.png"> |<img width="598" alt="Screenshot 2022-10-27 at 2 25 35 PM" src="https://user-images.githubusercontent.com/120486/198272458-0d4f1f99-e736-4121-8376-018d71c0c7ba.png"> |

Reply:
That's harder/more complex. And as this should be a low-traffic safety-net, I think it's not worth it. This should really be going away, soon. :crossed_fingers: